src/views/migration/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { PackageManifestKind, SubscriptionsKind } from './constant';
+
+export const createInstallUrl = (operator: PackageManifestKind, namespace?: string) =>
+  `/operatorhub/subscribe?pkg=${operator?.metadata?.name}&catalog=${operator?.status?.catalogSource}&catalogNamespace=${operator?.status?.catalogSourceNamespace}&targetNamespace=${namespace}`;
+
+export const createOperatorURL = (subscription: SubscriptionsKind) =>
```

Comment:
that does not look right, the url should look something like `mtv.apps.my-cluster.com` starting with `/k8s...` does not look good.

see:
https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator#obtaining-console-url_mtv